src/integrationTest/resources/application.yaml

```diff
@@ -15,24 +26,29 @@ spring:
     password: ${POSTGRES_PASSWORD:dbsyncdata}
 
 security:
-  authorisedServices:
-    - "it"
+  anonymousPaths:
+    - "/"
+    - "/health"
+    - "/health/liveness"
+    - "/actuator/**"
+    - "/loggers/**"
 ## IDAM feign connection
 idam:
+  api:
+    url: ${IDAM_URL:http://127.0.0.1:5000}
+    client-id: ${OAUTH2_CLIENT_ID:rd-professional-api}
```

Comment:
All these config values are required for actual token creation. Are we creating real token from SIDAM AAT in integration tests ?